README.rst

```diff
@@ -130,14 +133,14 @@ Creating A Plone Add-on
     $ plonecli -l
     Available mr.bob templates:
     - addon
-     - behavior
-     - content_type
-     - portlet
-     - theme
-     - theme_barceloneta
-     - view
-     - viewlet
-     - vocabulary
+    - behavior
```

Comment:
the indention is in fact what we want here, as these are sub-templates of the addon template ;)